### PR TITLE
Prime1: Logic Update

### PR DESCRIPTION
**Pull request by Samuel6710**
##### Impact Crater

- Added: The Metroid Prime Exoskeleton fight has full combat logic.

##### Chozo Ruins

- Added: Sun Tower Sessamoharu Complex Bomb Jump to Skip Super Missiles/Scan Visor

##### Phendrana Drifts

- Added: New Thardus Skip Method from Room Center
- Added: Quarantine Monitor to North Quarantine Tunnel Thardus Skip
- Added: Phendrana Shorelines Spider Track item without spider ball out of bounds trick

### CODE REVIEW

**Comment by Samuel6710** [comment]
The exo combat logic is definitely open to suggestions--you can likely refine it some more.

File: randovania/games/prime1/json_data/Phendrana Drifts.txt [hunk 1]
```diff
@@ -118,6 +118,9 @@ Extra - aabb: [-147.96088, -231.95529, -10.723184, 26.7513, -69.93009, 59.154648
               X-Ray Visor
   > Door to Plaza Walkway
       Trivial
+  > Pickup (Spider Track Missile)
+      # https://youtu.be/i1CJ_nBioSg
+      Boost Ball and Morph Ball and Scan Visor and Space Jump Boots and Combat/Scan Dash (Intermediate) and Single Room Out of Bounds (Advanced) and Standable Terrain (Advanced) and Shoot Super Missile
```
Comment (toasterparty) [inline randovania/games/prime1/json_data/Phendrana Drifts.txt:123]: and not door lock rando
and movement advanced (for the actual boost)

**Comment by Samuel6710** [comment]
??? I think I accidentally deleted stuff from changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -83,16 +72,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: When Progressive Grapple is enabled, it will now show `2 shuffled copies` rather than `Shuffled` for better consistency.
 - Changed: A proper error message is displayed when mono is not found, when exporting a game on macOS and Linux.
 
-#### Logic Database
```
Comment (toasterparty) [inline CHANGELOG.md:70]: don't delete stuff from changelog lol

File: randovania/games/prime1/json_data/header.json [hunk 3]
```diff
@@ -1092,7 +1092,7 @@
                                         "type": "resource",
                                         "data": {
                                             "type": "items",
-                                            "name": "Scan",
```
Comment (toasterparty) [inline randovania/games/prime1/json_data/header.json:1095]: ?

**Review by toasterparty — CHANGES_REQUESTED** [review]

**Comment by toasterparty** [comment]
For the PPC tricks, since you aren't adding videos now, make sure to update the spreadsheet to include those tricks if they don't already exist

**Comment by Samuel6710** [comment]
I think it's fine now?

File: randovania/games/prime1/json_data/header.json (hunk 3 above)
Comment (Samuel6710) [reply randovania/games/prime1/json_data/header.json:1095]: Can't tell what the issue is here exactly.

**Review by toasterparty — APPROVED** [review]

**Pull request merged**